plugins/obs-outputs/flv-output.c

```diff
@@ -41,12 +46,101 @@ struct flv_output {
 	bool sent_headers;
 	int64_t last_packet_ts;
 
+	enum video_id_t video_codec[MAX_OUTPUT_VIDEO_ENCODERS];
+
 	pthread_mutex_t mutex;
 
 	bool got_first_video;
 	int32_t start_dts_offset;
 };
 
+/* Adapted from FFmpeg's libavutil/pixfmt.h
+ *
+ * Renamed to make it apparent that these are not imported as this module does
+ * not use or link against FFmpeg.
+ */
+
+/* clang-format off */
+
+/**
+  * Chromaticity coordinates of the source primaries.
```

Comment:
```suggestion
  * Chromaticity coordinates of the source primaries
```
or
```suggestion
  * Define chromaticity coordinates of the source primaries
```

The first is more "label-like" or "header-like" and would be appropriate here.